README.rst

```diff
@@ -130,8 +130,8 @@ The repository consists mostly of externally hosted subrepositories:
 .. _protobuf: https://github.com/protocolbuffers/protobuf/
 .. |protobuflic| replace:: BSD-3 license
 .. _protobuflic: https://github.com/protocolbuffers/protobuf/blob/master/LICENSE
-.. |protobufver| replace:: 3.20.0
-.. _protobufver: https://github.com/protocolbuffers/protobuf/releases/tag/v3.20.0
+.. |protobufver| replace:: 21.1
```

Comment:
What a leap.

Reply:
Ironically not -- it's version 20 to version 21.  They dropped the semver 3.x-> 4.x prefix off the git branch for some reason.